content/events/devcon24/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "[Upcoming Event] Espressif DevCon24"
+date: 2024-06-11
+draft: false
+description: "Espressif Systems Developer Conference 24"
+tags: ["event", "announcement", "devcon"]
+showDate : false
+---
+
+{{< badge >}}
+New event!
+{{< /badge >}}
+
+## DevCon24
+
+Espressif DevCon24, the third edition of Developers conference made by Developers for Developers!
+
+{{< alert icon="eye">}}
+**The DevCon24, will take place online from September 3rd to 5th 13.00 to 18.00 CEST.**
```

Comment:
Suggest:

from September 3 to 5, 13.00 - 18.00 CEST

The way how we format dates and time should be consistent. See also my suggestions for the other event.